src/parser.rs

```diff
@@ -2526,9 +2526,15 @@ impl<'a> Parser<'a> {
         } else if self.parse_keywords(&[Keyword::ON, Keyword::UPDATE])
             && dialect_of!(self is MySqlDialect)
         {
-            Ok(Some(ColumnOption::DialectSpecific(vec![
-                Token::make_keyword("ON UPDATE"),
-            ])))
+            if self.parse_keyword(Keyword::CURRENT_TIMESTAMP) {
```

Comment:
Why did it change?

It's not related to your PR, so shouldn't have changed at all

Reply:
I mentioned it in my [first comment](https://github.com/sqlparser-rs/sqlparser-rs/pull/602#issue-1361406359).